scripts/benchmarks/README.md

```diff
@@ -92,3 +92,102 @@ python benchmark.py --help
 
 Note:
 - in `run_benchmarks.sh` we will clear the `RESULT_DIR` if it exists, to avoid contaimination with old results. To protect against overwrite, then always run with `NO_OVERWRITE=true`.
+
+## Logging Memory
+
+There are 2 ways to benchmark memory in `benchmark.py`:
```

Comment:
the recommended way to run benchmarks is via `tox` which calls the `benchmark.sh`, most people will not call `benchmark.py` directly
- so if benchmark.sh is hardcoded to a single mode, we need to justify why, based on the study here.
- and these instructions is only for advanced users who will run `benchmark.py` directly.